proj/src/main/webapp/chat.js

```diff
@@ -1001,6 +1052,15 @@ function addFriendToDom(uid) {
             })
         }
 
+        // sub to notifications
+        notifications.subscribeToTagChatId(tag, chatId);
```

Comment:
I think this needs to be a couple lines lower. `tag` is not defined until a couple lines later.